config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml

```diff
@@ -155,6 +155,10 @@ spec:
                   - type
                   type: object
                 type: array
+              managedPoliciesNs:
+                additionalProperties:
+                  type: string
+                type: object
               placementBindings:
```

Comment:
This status is "disconnected" from the list of managed policies below. Would it make sense to keep the namespaces associated with the policies from which they were derived? For example, making the "policies" status an object:
```
status:
  policies:
     - name: policy1
       ns: policy1NS
     - name: policy2
       ns: policy2NS
```

Reply:
This list is actually a list of the copied policies. I could make that policies status an object and include the name of the copied policy and the namespace of the parent policy. To be honest, I think the current approach is a bit cleaner, having the managed policies together with their namespaces.

Reply:
Maybe we can do a walk-through of the status and make any improvements we identify in a followup PR?